media/CircleGraph/index.js

```diff
@@ -147,6 +147,10 @@ host.BrowserHost = class {
     });
 
     this.window.addEventListener("message", (event) => {
+      if (window.origin !== event.origin) {
+        console.log("Unexpected Origin: ", event.origin);
+        return;
+      }
```

Comment:
@seanshpark I noticed that there is a potential security problem from `Verify the origin of the received message`.

I assumed that the messages are coming from the window's origin, would it be okay for CircleGraph?

Reply:
I'm not sure about technical background of this, but I think `event` itself would be coming from vscode process(? or thread?, the node-js).
And not sure about the fix but if this no problem loading the model, I think it would be OK.

Reply:
I agree this changes because this is a similar solution from sonarsource.com.

https://rules.sonarsource.com/javascript/RSPEC-2819/
```
window.addEventListener("message", function(event) {

  if (event.origin !== "http://example.org") // Compliant
    return;

  console.log(event.data)
});
```

Reply:
@jyoungyun I actually referred that solution :-D